interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-Researched answer:
+Researched answer: A conditional statement uses if to specify a block of code to be executed, if a specified condition is true, in order to continue.
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is the technology that GitHub, a code-sharing platform, uses. Git technology uses terminal to move code around. 
 
-Researched answer:
+Researched answer: Git is a tool used for managing code. Git is used to tracking changes and allowing for multiple coders to work on the same code. GitHub uses Git technology which is an online software development platform.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: An example of a JavaScript operator that will return a Boolean value is math problems with loose equality symbols.
 
-Researched answer:
+Researched answer: JavaScript operators that will return a Boolean value are those that use logical operators like AND and OR. 
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index is the locatiion of the variable in  an array. This starts at 0 and goes on infinitely. The difference between an index and a value is that a value is the numeric data that can mean anything. 
 
-Researched answer:
+Researched answer:An index is the locatiion of the variable in an array going from left to right. This can be either a number or letter and starts at 0 and goes on infinitely. The difference between an index and a value is that a value is a data chunk that represent pieces of information. 
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: An array is a list of variables such as names or numbers. A string is a one or more elements enclosed in quotation marks. They are both forms of data and can be manipulated using code. 
 
-Researched answer:
+Researched answer: An array is used to store multiple values in a single variable while a string is a variable that consists of one or more letters, numbers, or symbols. They are similar because they contain data that can be manipulated by code. 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-Your answer:
+Your answer: Pair programming is a technique in which two coders are working on the same coding project. There is the "driver" who is the one typing out the code in accordance to the "navigator" who is deciding on the final code. Both coders are collaborating with eachout though on ideas for the code. 
 
-Researched answer:
+Researched answer: Pair programming is a technique in which two programmers work together on one coding project. The driver writes code while the other, the navigator reviews each line of code as it is typed. Occasioanlly, the two coders switch roles.
```

Comment:
Good job!  I want to encourage you to really expand on your interview answers.  Think of these questions as if you are in a live interview and you want to show the employer your knowledge on the subject.  Examples are always a good way to showcase your understanding.